gui/qt/main_window.py

```diff
@@ -1245,7 +1259,8 @@ def do_update_fee(self):
                 _type, addr = self.get_payto_or_dummy()
                 outputs = [(_type, addr, amount)]
             try:
-                tx = self.wallet.make_unsigned_transaction(self.get_coins(), outputs, self.config, fee)
+                opreturn_message = self.message_opreturn_e.text().encode('utf-8').hex()
+                tx = self.wallet.make_unsigned_transaction(self.get_coins(), outputs, self.config, fee, None, opreturn_message) 
```

Comment:
Any reason the opreturn is not in the outputs list?  It is an output after all.

Reply:
considered doing it that way. this way seemed more straightfoward due to other things the outputs have to do... which reminds me i forgot to bump fee estimate.

Reply:
I had same suggestion as @kyuupichan about including it as an output, but given you can only have one op return output in a tx, wasn't 100% on it.  It'd be a lot cleaner though and future proof for when the bch devs remove this and other abitrary core dictated rules, and multiple ones can be added.